Stack/Opc.Ua.Core/Types/Encoders/BinaryEncoder.cs

```diff
@@ -88,7 +88,10 @@ protected virtual void Dispose(bool disposing)
                     m_writer.Dispose();
                 }
 
-                m_ostrm?.Dispose();
+                if (!m_leaveOpen)
```

Comment:
This should fix it.  The rest looks like formatting, so not doing a detailed compare, unless you need me to.

Reply:
what happened in this file? I need to revert the formatting